docroot/modules/custom/ccom_core/ccom_core.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains ccom_core.module.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ccom_core_preprocess_block(&$variables) {
+  if (isset($variables['attributes']['id']) && $variables['attributes']['id'] === 'block-uids-base-footercontactinfo') {
```

Comment:
```suggestion
  if (isset($variables['attributes']['id']) && str_contains($variables['attributes']['id'], 'footercontactinfo')) {
```

Looks like we have a mess of block ids and they aren't always the same. Noticed the id on sandbox was `block-uids-base-footercontactinfo--2`